typings/index.d.ts

```diff
@@ -96,3 +96,6 @@ export interface NetInfoStatic {
    */
   isConnectionExpensive: () => Promise<boolean>;
 }
+
+declare let NetInfo: NetInfoStatic;
```

Comment:
Not sure it matters but should these be inside a module declaration? 

**Example**: https://github.com/invertase/react-native-firebase/blob/master/packages/analytics/lib/index.d.ts#L111-L137

Reply:
I think that that is only needed if you want to define multiple module definitions in a single file. As this file is for the entire module as a whole it is not needed.

https://www.typescriptlang.org/docs/handbook/modules.html#ambient-modules